datasets/Attai_2022/compose.py

```diff
@@ -0,0 +1,24 @@
+import pandas as pd
+import sys
+
+sys.path.append("../../scripts")
+import utils
+
+# Read input
+sheets = pd.read_excel(
+    "https://zenodo.org/records/7243308/files/Systematic_Literature_Review_Data.xlsx?download=1",
+    engine="openpyxl",
+    sheet_name=None,
+)
+
+search = sheets["Exclude_Papers"]
+ft = sheets["Included_Papers"]
+
+search = utils.rename_columns(search, title="Title")
+ft = utils.rename_columns(ft, title="Title", year="Year")
```

Comment:
I think the variable name `search` is incorrect, it's just the excluded papers right, not the full search? (nitpick, I know)